Content.Server/CartridgeLoader/Cartridges/SpaceVendorsCartridgeComponent.cs

```diff
@@ -0,0 +1,15 @@
+
+using Content.Shared.CartridgeLoader.Cartridges;
+
+namespace Content.Server.CartridgeLoader.Cartridges
+{
+    [RegisterComponent]
+    public sealed class SpaceVendorsCartridgeComponent : Component
+    {
+        [DataField("appraisedItems")]
+        public List<AppraisedItem> AppraisedItems = new();
+
+        [DataField("maxSavedItems")]
+        public int MaxSavedItems { get; set; } = 9;
+    }
+}
```

Comment:
1. file-scoped namespaces please.
2. VV readwrite.
3. Make the second one a field instead of a property.
4. Needs comments
5. Access attribute.

Reply:
VV readwrite means adding `ViewVariables(VVAccess.ReadWrite)]` beind the `DataField` attribute.

Reply:
I understood everything except 2 and 5. What does it mean VV readwrite and what is an attribute?

Reply:
Attributes are the tag like lines above classes, methods and properties.
`[DataField("appraisedItems")]` is an attribute for example and if someone tells you that you need to add VV readwrite they mean that you need to add the `[ViewVariables(VVAccess.ReadWrite)]` attribute.
You can add multiple attributes in one line by putting them all inbetween a single set of brackets and seperating them with a comma. 
Like this for example: `[DataField("appraisedItems"), ViewVariables(VVAccess.ReadWrite)]`.

Point 5 means that you need to add the access attribute above your class. The access attribute restricts what classes can modify the fields of your components. You generally want to restrict that to the system that deals with your component.
The access attribute looks like this: `[Access(typeof(<Your system goes here>))]` so you would need to write:
```cs
    [RegisterComponent, Access(typeof(SpaceVendorsCartridgeSystem))]
    public sealed class SpaceVendorsCartridgeComponent : Component
````

Here is the c# documentation for attributes:
https://learn.microsoft.com/en-us/dotnet/csharp/advanced-topics/reflection-and-attributes/
https://learn.microsoft.com/en-us/dotnet/csharp/advanced-topics/reflection-and-attributes/attribute-tutorial